src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/entity/timeline/TimelineEntity.kt

```diff
@@ -20,34 +19,30 @@ import java.util.UUID
 
 @Table(name = "timeline")
 @Entity
-class TimelineEntity(
-  @Id
-  @GeneratedValue
-  @UuidGenerator
-  var id: UUID? = null,
-
+data class TimelineEntity(
```

Comment:
I've taken the opportunity to refactor the entity class to be a data class, and to move the JPA managed fields (id, createdBy, createdAt etc) out of the constructor args and simply have them as fields.
We already have a couple of entity classes that follow this pattern, and it feels "cleaner", in that the constructor is the public API of this entity, and we would not expect consumers of this code who want to create the entity to have to provide the ID and createdBy/At audit fields (that is the job of the framework / JPA and the lifecycle event listeners)